src/Nextjs.ts

```diff
@@ -113,16 +121,24 @@ export class Nextjs extends Construct {
     // get dir to store temp build files in
     const tempBuildDir = props.tempBuildDir
       ? path.resolve(
-        path.join(props.tempBuildDir, `nextjs-cdk-build-${this.node.id}-${this.node.addr.substring(0, 4)}`)
-      )
+          path.join(props.tempBuildDir, `nextjs-cdk-build-${this.node.id}-${this.node.addr.substring(0, 4)}`)
+        )
       : fs.mkdtempSync(path.join(os.tmpdir(), 'nextjs-cdk-build-'));
 
     this.tempBuildDir = tempBuildDir;
 
     // create static asset bucket
     this.staticAssetBucket =
       props.defaults?.assetDeployment?.bucket ??
-      new s3.Bucket(this, 'Bucket', {
+      new s3.Bucket(this, 'AssetBucket', {
+        removalPolicy: RemovalPolicy.DESTROY,
+        autoDeleteObjects: true,
+      });
+
+    // create cache bucket
+    this.cacheBucket =
+      props.defaults?.assetDeployment?.bucket ??
```

Comment:
using "assetDeployment.bucket" might be misleading if we're separating out the buckets for assets and cache.

Reply:
Certainly agree. One thing I'm not sure about is whether we need additional deployment options. In https://github.com/kevin-mitchell/cdk-nextjs/blob/e60c3bfb5dc00d1871cea18de6ad3b91b29c1c79/src/Nextjs.ts#L32  there are all sorts of options about the deployment, but to be honest I'm not 100% sure how this works when re-deploying over something already in production. `cache` should be temporal I'd think, and safe to be destroyed, but I also know large production sites with tends of thousands of pages it can be extremely expensive to just throw away all cache. 

For now, for this PR I'll take the approach of "KISS" and just make a simple cache bucket unless somebody tells me otherwise. This would be something to improve on in another PR potentially. Obviously open to a better option though!